### PR TITLE
Make PortalID a Github Secret

### PR DESCRIPTION
**Pull request by mtndan**
**Types of change**

- [ ] Bug fix (change which fixes an issue)
- [ x ] Enhancement (change which improves upon an existing feature)
- [ ] New feature (change which adds new functionality)

**Description**

Rather than hardcoding the PortalID in the deploy script, I added a simple variable so we can have a Github Secret, much like the API key.

**Relevant links**

N/A

**Checklist**

- [x ] I have read the [CONTRIBUTING](https://github.com/HubSpot/cms-theme-boilerplate/blob/master/CONTRIBUTING.md) document.
- [ x] My code follows the [style guide requirements](https://github.com/HubSpot/cms-theme-boilerplate/blob/master/STYLEGUIDE.md).
- [ x] I have thoroughly tested my change.

**People to notify**

### CODE REVIEW

**Comment by gcorne** [comment]
@jasonnrosa someone will need to add the portal ID as a secret for this repo.

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
           npm install
     - name: Generate hubspot.config.yml and deploy
       env:
-        HUBSPOT_PORTAL_ID: 6326501
+        HUBSPOT_PORTAL_ID: ${{ secrets.HubSpotPortalID }}
```
Comment (gcorne) [inline .github/workflows/deploy.yml:20]: Minor thing, but we tend to use camel case for `ID` so `secrets.HubSpotPortalId`.

Comment (mtndan) [reply .github/workflows/deploy.yml]: Fixed!

**Comment by mtndan** [comment]
The purpose is to keep environment-specific data out of code. If I am going to use this project on my own, it makes sense to have secrets for both the Portal ID and the API Key - both are project/env specific, and not related to the functionality of the project itself.

--
Dan Katz
ElectricSage, LLC

Would you like to schedule a call?

**Pull request merged**